lib/Convos/Controller/Connection.pm

```diff
@@ -20,7 +20,13 @@ sub create {
   }
 
   eval {
-    $connection = $user->connection({protocol => $url->scheme, name => $name});
+    $connection = $user->connection(
+      {
+        name                => $name,
+        on_connect_commands => $json->{on_connect_commands} || [],
```

Comment:
I think we should do some sanitising on the input. Something like this:

```
@cmds = grep {/\w/} @{$json->{on_connect_commands} || []};
```

We could even do that in the schema, but I'm not sure what is best...